tests/local/e2e_test.go

```diff
@@ -142,6 +142,11 @@ var _ = ginkgo.Describe("[Teleporter integration tests]", func() {
 		func() {
 			flows.ResubmitAlteredMessage(LocalNetworkInstance)
 		})
+	ginkgo.It("Check upgrade access",
+		ginkgo.Label("upgradeability"),
```

Comment:
I missed my chance to comment this on https://github.com/ava-labs/teleporter/pull/273 before it was merged, but can we define string consts for these labels to reduce the chances of a frustrating to debug typo?